### DISCUSSION TITLE
How does the /save command actually work?

### DISCUSSION THREAD

**Discussion by eWOOD29** [ProfSynapse/Professor-Synapse#5] [Q&A] [answered]
I understand that the /save command summarizes progress and recommends next steps, but how does it actually extend context limits? I seem to be running into context limits fairly quickly, even when using the /save command, and I'm wondering if the information provided from the use of the /save command could be given to another instance to continue where the previous one left off? Just looking for some insight into how the /save command actually achieves its purpose so that I could better implement it myself in the future.

**Comment by sharpie12** [comment, marked as answer]
It does not, no persona or prompt can extend the context window. You save once you believe you are at or near the window, and then copy and paste the results into a new session. Edit to add that the wording for /save is a bit misleading on the front page. The context window is scrolling memory (sorry if you know this already), it deletes whatever lines are oldest as you keep interacting. when you “save” it is summarizing your progress and so it seems like it is going past the window a bit, but in reality it is just using the summarization and context clues from whatever is still in its window. No way around it.

**Reply by eWOOD29** [reply]
Thank you for the answer, I appreciate you taking the time!
